doc/quickfixsigns.txt

```diff
@@ -437,7 +437,7 @@ g:quickfixsigns#vcsdiff#vcs    (default: {...})
         cmd ... command templates that generate a unified diff file. 
         "%s" is replaced with the filename.
         dir ... the directory name
-        revision ... The default revision/branch
+        rev_arg ... argument to selection revision to diff against
```

Comment:
Thanks for updating the help file. The help file is auto-generated from time to time so it's not absolutely necessary.